client/cli/src/params/pruning_params.rs

```diff
@@ -32,7 +32,7 @@ pub struct PruningParams {
 	pub state_pruning: Option<String>,
 	/// Specify the number of finalized blocks to keep in the database.
 	///
-	/// Default is to keep all blocks.
+	/// Default is to keep all of finalized blocks.
```

Comment:
Here we should add that 0 keep all blocks.
Generally this is not very intuitive.
Maybe we should do something like `state_pruning`: use the same string: 'archive' for 'keep_all' and 'final' fo 'keep_finalized'. (subject to change of name too).

Reply:
Hmm,,It's really not very intuitive that 0 keep all blocks. IMO it's a  compromise approach now.
I agree that change `blocks_pruning` to be like `state_pruning`.